diff --git a/BUILD.bazel b/BUILD.bazel
index 7b5bc3ee..566eb0c4 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -43,6 +43,7 @@ filegroup(
     name = "android_srcs",
     srcs = glob([
         "compose-app/**",
+        "compose-resources/**",
         "presentation-core/**",
         "presentation-feature/**",
         "presentation-model/**",
@@ -64,6 +65,7 @@ filegroup(
     name = "desktop_srcs",
     srcs = glob([
         "compose-app/**",
+        "compose-resources/**",
         "presentation-core/**",
         "presentation-feature/**",
         "presentation-model/**",
@@ -84,6 +86,7 @@ filegroup(
     name = "ios_framework_srcs",
     srcs = glob([
         "compose-app/**",
+        "compose-resources/**",
         "presentation-core/**",
         "presentation-feature/**",
         "presentation-model/**",
